tests/test_vector2_rotate.py

```diff
@@ -15,20 +15,25 @@
     (Vector2(math.pi, math.e), 67, Vector2(-1.27467, 3.95397))
 ]
 
-@pytest.mark.parametrize('input, degrees, expected', data)
-def test_multiple_rotations(input, degrees, expected):
-    assert input.rotate(degrees).isclose(expected)
-    assert angle_isclose(input.angle(expected), degrees)
+@pytest.mark.parametrize('input, angle, expected', data)
```

Comment:
I don't know if I'd have called this one a necessary change, but I feel like it's more clear what you're going for here.